src/components/DevNavMenu.tsx

```diff
@@ -540,24 +540,7 @@ DevNavMenuTuts.set('firstChain', [
 DevNavMenuTuts.set('poe', [
   {
     name: `Build a Proof of Existence Blockchain`,
-    items: [
-      {
-        title: `Introduction`,
-        link: '/tutorials/v3/proof-of-existence/',
-      },
-      {
-        title: `Preparation`,
-        link: '#prepare-to-build-a-dapp',
-      },
-      {
-        title: `Build a Custom Pallet`,
-        link: '#building-a-custom-pallet',
-      },
-      {
-        title: `Build a Custom Front-end`,
-        link: '#building-a-custom-front-end',
-      },
-    ],
+    items: [ ],
```

Comment:
does this render as a dropdown still?

Reply:
The modal does but it doesn't do anything. Hack for now until we figure out new UX and restructuring going forward. 🤷🏻